.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,28 @@
+name: "Create release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Create release'
+        default: ""
+        required: true
+
+jobs:
+  publish-release:
+    name: Publish release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Wait for tests to succeed
+        uses: lewagon/wait-on-check-action@v1.3.1
```

Comment:
It's a good practice to use the sha instead of the version for 3rd party GitHub Actions.
Additional Info can be found here: https://michaelheap.com/ensure-github-actions-pinned-sha/